lxd/migrate.go

```diff
@@ -512,45 +512,80 @@ func (s *migrationSourceWs) Do(migrateOp *operation) error {
 }
 
 type migrationSink struct {
-	migrationFields
-
-	url    string
-	dialer websocket.Dialer
+	// We are pulling the container from src in pull mode.
```

Comment:
My understanding is that push mode is copying from `src` to `dest`, i.e. the operation is still pulling data from `src`.  This comment implies that `src` is only used in pull mode.